.github/workflows/dependabot-auto-merge.yml

```diff
@@ -0,0 +1,28 @@
+# dependabot[bot]
+
+name: Dependabot Auto Merge
+
+on:
```

Comment:
Where is the filter for the "dependabot" label?

Reply:
I think it relies on author:

> GITHUB_LOGIN: dependabot[bot]

Reply:
Yep, ridedott/merge-me-action checks for author. I tried adding a label check as well but it didn't work as expected for some reason. I believe the author check should be good enough.